scripts/scrape_mercury_benchmarking_efficiency_data.py

```diff
@@ -49,15 +50,29 @@ def parse(runs, keys):
             all_data.append((run, *[data[key] for key in keys]))
         except Exception as e:
             print(f"Could not parse metrics for run {run} at {url}")
-            print(r.text)
             continue
 
     return all_data
 
 
+def save_per_instance_metrics(runs):
```

Comment:
Maybe to match this format we should combine the `parse` and `save` functions into a single `parse_and_save_aggregated_metrics` function or something like that

Reply:
Or add a third command line argument for `--parse_aggregated_metrics`, and then `--save_aggregated_metrics` will just download the files without parsing as in `save_per_instance_metrics`